src/main/resources/static/err.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Nie dodano użytkownika</title>
+</head>
+<body>
+    <h1>Użytkownik nie został dodany, wprowadź imię.</h1>
+    <h2><a href="http://localhost:8080/">Powrót do formularza</a></h2>
```

Comment:
Nie używaj adresów absolutnych. Jeżeli np. zmienisz port, na którym działa aplikacja z 8080 na 8081, to automatycznie ten kod przestanie działać. Zamiast tego wystarczy "/"